Slim/Player/TranscodingHelper.pm

```diff
@@ -133,7 +133,10 @@ sub loadConversionTables {
 # I - can transcode from stdin
 # F - can transcode from a named file
 # R - can transcode from a remote URL (URL types unspecified)
-# H - request to strip out header (wav/aif only)
+#
```

Comment:
Hmm... that would be backwards breaking, wouldn't it? I don't think we use it in the stock convert.conf. But I'm sure some audiophile will grieve for the loss of his meticulously crafted custom-convert.conf rules...

Reply:
Yes, it breaks backward compatibility. I can always keep the 'H' option in addition, but I think it's very unlikely that it is used currently and probably now is the best timing to not carry 'bagage'. If we change it and somebody cries, it's easy to put it back. It also depends if you want to push it as a 8.0.1 or a 8.1 :-)